content/en/blog/2022/frontend-overhaul/index.md

```diff
@@ -0,0 +1,181 @@
+which are used to show how to instrument a distributed system with
+OpenTelemetry.
+
+The main focus of the web app is to demonstrate how to instrument an
+application, no matter what programming language, platform or OS it uses. The
+web app also shows different instrumentation techniques: automatic and manual,
+metrics, and baggage. All while following the standards and conventions
+prescribed in the official OTel documentation. More about the specific
+requirements can be
+[found here](https://github.com/open-telemetry/opentelemetry-demo/tree/main/docs/requirements).
+
+My company was focused on becoming part of and embracing the OpenTelemetry
+community. One of our goals this summer was to get more involved with a core
+OpenTelemetry project where we could provide a meaningful contribution. The OTel
+demo was the best match for our goal, as contributing would not only help the
+community, but also provide a great example to test and showcase our product
+with.
+
+The first thing we did was to get in contact with
+[Carter Socha](https://github.com/cartersocha), the organizer of the OTel Demo
+SIG. Carter was really welcoming and helped us identify where our contributions
+could be the most impactful. We started looking at
+[the issue created by Austin Parker](https://github.com/open-telemetry/opentelemetry-demo/issues/39)
+referencing a complete front-end overhaul that would involve moving the
+application away from Go server-side render (SSR) to an architecture that
+included a browser-side client (client-side render or CSR), as well as improving
+the overall style, theme, and user experience.
+
+A fun aspect of the work was the request to move the store from a "normal" store
+to an astronomy store to match the OpenTelemetry project’s overall branding.
+
+Once we got the green light from the rest of the OTel demo SIG, then we started
+working on the different changes that were part of the front-end architecture
+overhaul.
+
+## OpenTelemetry Demo Application Description and Tech Stack
+
+The demo app is an astronomy store, with basic eCommerce features such as a
+shopping cart, currency selector, payment, and checkout. It also includes the
+ability to display promotions (ads) and related products depending on the user’s
+context.
+
+The demo’s stack includes multiple microservices in different languages,
+covering each of the following OTel-supported languages:
+
+- [.NET](https://opentelemetry.io/docs/instrumentation/net/)
+- [C++](https://opentelemetry.io/docs/instrumentation/cpp/)
+- [Erlang/Elixir](https://opentelemetry.io/docs/instrumentation/erlang/)
+- [Go](https://opentelemetry.io/docs/instrumentation/go/)
+- [Node.js](https://opentelemetry.io/docs/instrumentation/js/)
+- [PHP](https://opentelemetry.io/docs/instrumentation/php/)
+- [Python](https://opentelemetry.io/docs/instrumentation/python/)
+- [Ruby](https://opentelemetry.io/docs/instrumentation/ruby/)
+- [Rust](https://opentelemetry.io/docs/instrumentation/rust/)
```

Comment:
Changes:

- Use local paths not external URLs here
- Alphabetical order of languge names: put .NET after C++ (as if it were named dotnet).

```suggestion
- [C++](/docs/instrumentation/cpp/)
- [.NET](/docs/instrumentation/net/)
- [Erlang/Elixir](/docs/instrumentation/erlang/)
- [Go](/docs/instrumentation/go/)
- [Node.js](/docs/instrumentation/js/)
- [PHP](/docs/instrumentation/php/)
- [Python](/docs/instrumentation/python/)
- [Ruby](/docs/instrumentation/ruby/)
- [Rust](/docs/instrumentation/rust/)
```